src/content/home-page.json

```diff
@@ -9,11 +9,14 @@
 		"vault": {
 			"cardTitle": "Vault",
 			"url": "/vault/tutorials/cloud/vault-introduction",
-			"description": "Access Vault’s secrets management and encryption capabilities instantly using the HCP portal",
+			"description": "Access Vault’s secrets management and encryption capabilities instantly using the HCP portal.",
 			"ctaText": "Learn more"
 		},
-		"hashiconfGlobal": {
-			"description": "Two days of inspirational speakers, community connection, and a front-row seat to future of cloud."
+		"hcp": {
+			"cardTitle": "HashiCorp Cloud Platform",
+			"url": "https://portal.cloud.hashicorp.com/sign-up",
```

Comment:
I think this is the right URL, based on this Asana task: 🎟️ [Decide on activity to highlight](https://app.asana.com/0/0/1203011399855183/f)